index.js

```diff
@@ -3,64 +3,23 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var restify = require('restify')
-var error = require('../error')
-var bufferize = require('../bufferize')
-var config = require('../config')
-var log = require('../log')(config.logLevel, 'db-api')
-var DB = require('../db/mysql')(log, error)
-var pkg = require('../package.json')
+var bufferize = require('./bufferize')
+var version = require('./package.json').version
 
-var MemoryMonitor = require('../memory_monitor')
-var memoryMonitor = new MemoryMonitor()
```

Comment:
This MemoryMonitor stuff is no longer in any of the three repos.  Do we want to keep it?  AFAICT it's no longer uses in the fxa-auth-server so maybe not.